resident/resident-service/src/test/java/io/mosip/resident/test/controller/ResidentCredentialControllerTest.java

```diff
@@ -18,135 +29,141 @@
+    @Test
+    public void testCancelRequestSuccess() throws Exception {
 
-	}
+        Mockito.when(residentCredentialService.cancelCredentialRequest(Mockito.any()))
+                .thenReturn(credentialCancelReqResponse);
 
-	@Test
-	public void testCancelRequestSuccess() throws Exception {
+        mockMvc.perform(MockMvcRequestBuilders.get("/req/credential/cancel/requestId")
+                        .contentType(MediaType.APPLICATION_JSON_VALUE))
+                .andExpect(status().isOk());
 
-		Mockito.when(residentCredentialService.cancelCredentialRequest(Mockito.any()))
-				.thenReturn(credentialCancelReqResponse);
+    }
 
-		mockMvc.perform(MockMvcRequestBuilders.get("/req/credential/cancel/requestId")
-				.contentType(MediaType.APPLICATION_JSON_VALUE))
-				.andExpect(status().isOk());
+    @Test
+    public void testgetCredentialRequestStatusSuccess() throws Exception {
 
-	}
+        Mockito.when(residentCredentialService.getStatus(Mockito.any())).thenReturn(credentialReqStatusResponse);
 
-	@Test
-	public void testgetCredentialRequestStatusSuccess() throws Exception {
+        mockMvc.perform(MockMvcRequestBuilders.get("/req/credential/status/requestId")
+                        .contentType(MediaType.APPLICATION_JSON_VALUE))
+                .andExpect(status().isOk());
 
-		Mockito.when(residentCredentialService.getStatus(Mockito.any())).thenReturn(credentialReqStatusResponse);
+    }
 
-		mockMvc.perform(MockMvcRequestBuilders.get("/req/credential/status/requestId")
-				.contentType(MediaType.APPLICATION_JSON_VALUE))
-				.andExpect(status().isOk());
+    @Test
+    public void testgGetCardSuccess() throws Exception {
 
-	}
+        Mockito.when(residentCredentialService.getCard(Mockito.any())).thenReturn(pdfbytes);
 
-	@Test
-	public void testgGetCardSuccess() throws Exception {
+        mockMvc.perform(MockMvcRequestBuilders.get("/req/card/requestId")
+                .contentType(MediaType.APPLICATION_JSON_VALUE)).andExpect(status().isOk());
 
-		Mockito.when(residentCredentialService.getCard(Mockito.any())).thenReturn(pdfbytes);
+    }
 
-		mockMvc.perform(MockMvcRequestBuilders.get("/req/card/requestId")
-				.contentType(MediaType.APPLICATION_JSON_VALUE)).andExpect(status().isOk());
+    @Test
+    public void testGetCredentialTypesSuccess() throws Exception {
 
-	}
+        mockMvc.perform(MockMvcRequestBuilders.get("/credential/types").contentType(MediaType.APPLICATION_JSON_VALUE))
+                .andExpect(status().isOk());
 
-	@Test
-	public void testGetCredentialTypesSuccess() throws Exception {
+    }
 
-		mockMvc.perform(MockMvcRequestBuilders.get("/credential/types").contentType(MediaType.APPLICATION_JSON_VALUE))
-				.andExpect(status().isOk());
+    @Test
+    public void testPartnerIdCredentialType() throws Exception {
 
-	}
+        mockMvc.perform(MockMvcRequestBuilders.get("/req/policy/partnerId/1/credentialType/credentialType").contentType(MediaType.APPLICATION_JSON_VALUE))
```

Comment:
the last credentialType in the URL is not a valid type.
Can we do more assertions with the response like errors are empty ?

Reply:
@abuabdul As discussed, lets add it to the sub task and merging it for now